webapp/src/js/enketo/bootstrap-datepicker.tl.js

```diff
@@ -2,17 +2,30 @@
 /**
```

Comment:
Do we care about the blank line 1?

Reply:
Sorry, I don't think I understand what you mean.